lib/plugins/aws/package/compile/events/websockets/index.js

```diff
@@ -51,9 +51,45 @@ class AwsCompileWebsockets {
       },
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8019
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'websocket', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            route: { type: 'string' },
+            routeResponseSelectionExpression: {
+              type: 'string',
+              regexp: /^(?:\$default)$/.toString(),
+            },
+            authorizer: {
+              anyOf: [
+                { type: 'string' },
```

Comment:
I think it'll be good to narrow this, to two specific _string_ types we actually support here:

1. `awsArnString`, we have it defined here https://github.com/serverless/serverless/blob/ce51c8fb6fea254affd51361b6a1cf551b5d8a36/lib/plugins/aws/provider/awsProvider.js#L150
2. `functionName`, which is expressed by this pattern:https://github.com/serverless/serverless/blob/ce51c8fb6fea254affd51361b6a1cf551b5d8a36/lib/configSchema.js#L3 I would add `functionName` definition here: https://github.com/serverless/serverless/blob/ce51c8fb6fea254affd51361b6a1cf551b5d8a36/lib/configSchema.js#L114 and reuse it

Reply:
done! I added `functionName` to the `definitions` in the schema but I also kept the `functionNamePattern` variable cause it's used for a pattern property somewhere else in the schema